website/docs/tutorials/06-deploying/02-migrations/README.md

```diff
@@ -5,21 +5,18 @@ sidebar_label: Migrations
 # Migrations
 
 :::warning
-This feature is actively developed. Many new additions or changes will land in future Protostar releases.
+Breaking changes can be introduced without depreciation.
```

Comment:
i liked the old phrasing better - or we can do a bit of both. We want to tell the users that it's actively developed, and to expect breaking changes

Reply:
![Screen Shot 2022-10-04 at 18 06 47](https://user-images.githubusercontent.com/9629577/193869684-bbb157dd-4550-4f4a-b2a7-850d41e0b570.png)
https://www.youtube.com/watch?v=3duFa_5Zq6A

Reply:
Can't argue with that